api/pyproject.toml

```diff
@@ -12,6 +12,7 @@ uvicorn = "^0.24.0.post1"
 channels = "^4.0.0"
 psycopg2-binary = "^2.9.9"
 django-environ = "^0.11.2"
+django-ninja = "^1.0.1"
```

Comment:
**question:** Should it be 1.1.0? https://github.com/vitalik/django-ninja/releases